README.md

```diff
@@ -351,6 +352,47 @@ Warehouse has the ability to use [passport-npm] to check authorization when
 connecting via `npm`. An example of this can be found in the
 [tests for npm auth][auth].
 
+## Getting a package `warehouse.ai` - ready
```

Comment:
Should this cover how to setup auth for scopes, etc?  It should probably list some assumptions about the setup.